lodash-plugin/src/test/java/com/github/underscore/lodash/LodashTest.java

```diff
@@ -669,6 +669,8 @@ public void chain() {
         $.chain(new Integer[] {0}).min(new Function1<Integer, Integer>() {
             public Integer apply(Integer value) { return value; } });
         $.chain(new Integer[] {0}).sort();
+        $.chain(new Integer[] {0}).sortWith(new Comparator<Integer>() {
```

Comment:
\[FindBugs\] INFO: RV: Return value of $$Chain\.sortWith\(Comparator\) ignored, is this OK in com\.github\.underscore\.lodash\.LodashTest\.chain\(\)